File: 485-MaxConsecutiveOnes/485-MaxConsecutiveOnes.java
// Last updated: 31/07/2025, 21:16:11
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int c=0,m=0;
        for(int ele :nums)
        {
            if(ele==1)
            {
                c+=1;
                m=Math.max(m,c);
            }
            else
            {
                c=0;
            }
        }
        return m;
        
    }
}

File: 485-MaxConsecutiveOnes/485-MaxConsecutiveOnes.py
# Last updated: 31/07/2025, 21:16:16
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        c=0
        m=0
        for i in nums:
            if i==1:
                c+=1
                m=max(m,c)
            else:
                c=0
        return m            
                    

        